pypln/web/backend_adapter/pipelines.py

```diff
@@ -44,6 +44,10 @@ def create_indexing_pipeline(doc):
             {"index_name": doc.index_name, "doc_type": doc.doc_type}})
     (Extractor().si(doc_id) | ElasticIndexer().si(doc_id))()
 
+def corpus_freqdist(corpus):
```

Comment:
Shouldn't this have a verb-name, like ```calculate_corpus_freqdist```? I thought the call on the view was strange because of this.